src/TicTacToe.js

```diff
@@ -26,18 +27,19 @@ TicTacToe.prototype.determineGameType = function(e) {
 }
 
 TicTacToe.prototype.playerVsPlayerGame = function() {
-  this.ui = new UI;
+  this.ui = new UI();
   this.GameBoard = new GameBoard();
   this.WinConditions = new WinConditions();
   this.game = new PlayerVsPlayerGame(this.ui, this.GameBoard, this.WinConditions);
   this.game.initializeGame();
 }
 
 TicTacToe.prototype.playerVsComputerGame = function() {
-  this.ui = new UI;
+  this.ComputerLogic = new ComputerLogic();
+  this.ui = new UI();
```

Comment:
you seem to define `this.ui` as the same thing in every method, any reason for this?